docs/operations/operations-import-ALZAccount.md

```diff
@@ -0,0 +1,206 @@
+# Operations - How to migrate an Amazon Landing Zone (ALZ) account "as is" into an Amazon Secure Environment Accelerator.
```

Comment:
Can you change Amazon to AWS in both spots?

Reply:
Done - made the changes  and had one commit with spelling, grammar, added some more subtitles and tried to simplify where possible.   Please feel free to edit changes directly.